.experimenter.yaml

```diff
@@ -59,7 +59,7 @@ nimbus-validation:
       type: string
       description: The title of displayed in the Settings screen and app menu.
 onboarding:
-  description: The onboarding features displayed on homescreen.
+  description: The homescreen that the user goes to when they press home or new tab.
```

Comment:
At the moment, onboarding is shown once, after user dismiss it they don't see it anymore

Reply:
I realize why this is weird now. It's because the description is being replaced by the description in https://searchfox.org/mozilla-mobile/source/fenix/nimbus.fml.yaml#37

Reply:
https://searchfox.org/mozilla-mobile/source/fenix/.experimenter.yaml#62 is where the original onboarding description is coming from